packages/shared/src/components/cards/RaisedLabel.tsx

```diff
@@ -1,41 +1,51 @@
 import React, { ReactElement } from 'react';
 import classNames from 'classnames';
-import styles from './Card.module.css';
 import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+import classed from '../../lib/classed';
 
 export enum RaisedLabelType {
   Hot = 'Hot',
   Pinned = 'Pinned',
+  Beta = 'Beta',
 }
 
 const typeToClassName: Record<RaisedLabelType, string> = {
   [RaisedLabelType.Hot]: 'bg-theme-status-error',
   [RaisedLabelType.Pinned]: 'bg-theme-bg-bun',
+  [RaisedLabelType.Beta]: 'bg-cabbage-40',
 };
 
 export interface RaisedLabelProps {
   listMode?: boolean;
   type: RaisedLabelType;
   description?: string;
+  className?: string | undefined;
 }
 
+export const RaisedLabelContainer = classed(
+  'div',
+  'temp relative group group-hover:transform-none group-hover:z-10 group-hover:transition-transform group-hover:ease-linear group-hover:transition-z-index group-hover:duration-100 group-hover:step-end',
+);
+
 export function RaisedLabel({
   listMode,
   type = RaisedLabelType.Hot,
   description,
+  className,
 }: RaisedLabelProps): ReactElement {
   return (
     <div
       className={classNames(
         'absolute flex items-start',
         listMode ? 'top-0 right-full mt-5' : 'left-0 bottom-full ml-5 h-5',
+        className,
       )}
     >
       <SimpleTooltip content={description}>
         <div
           className={classNames(
             'flex items-center px-1',
-            styles.flag,
+            'will-change-auto -z-1 transition-transform duration-100 ease-linear group-hover:transform-none group-hover:z-10 group-hover:transition-transform group-hover:ease-linear group-hover:transition-z-index group-hover:duration-100 group-hover:step-end',
```

Comment:
[non-actionable] Wow, we were able to translate it but it did become a huge chunk of classes. I wonder how we can name this better. Though, still the goal is to lessen the bundle by reducing CSS modules that bloat the app. Well done 🚢